src/screens/loading.rs

```diff
@@ -22,7 +22,10 @@ fn show_loading_screen(mut commands: Commands) {
         .ui_root()
         .insert(StateScoped(Screen::Loading))
         .with_children(|children| {
-            children.label("Loading...");
+            children.label("Loading...").insert(Style {
+                justify_content: JustifyContent::Center,
+                ..default()
+            });
```

Comment:
I don't think this does anything? This overwrites the initial value of:
```rust

                style: Style {
                    width: Px(500.0),
                    justify_content: JustifyContent::Center,
                    align_items: AlignItems::Center,
                    ..default()
                },
```
Shouldn't that already be centering the text?